jest.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   ],
   moduleNameMapper: {
     '\\.(svg|png)$': '<rootDir>/empty-module.js',
+    '^@crayons(.*)$': '<rootDir>/app/javascript/crayons$1',
```

Comment:
Allows us to use the webpack alias `@acrayons` in jest as well.

Reply:
what `jest` stands for? (in file name)... just out of curiosity..

Reply:
This is the configuration file for the testing framework we use in the frontend. See https://jestjs.io